lib/singlepass-backend/src/lib.rs

```diff
@@ -17,8 +17,10 @@
     all(target_os = "macos", target_arch = "x86_64"),
     all(target_os = "linux", target_arch = "x86_64"),
     all(target_os = "linux", target_arch = "aarch64"),
+    all(target_os = "android", target_arch = "x86_64"),
+    all(target_os = "android", target_arch = "aarch64"),
 )))]
-compile_error!("This crate doesn't yet support compiling on operating systems other than FreeBSD, linux and macos and architectures other than x86_64");
+compile_error!("This crate doesn't yet support compiling on operating systems and architectures other than these:\n       - FreeBSD and x86_64\n       - FreeBSD and AArch64\n       - macOS and x86_64\n       - Linux and x86_64\n       - Linux and AArch64\n       - Android and x86_64\n       - Android and AArch64");
```

Comment:
I'd prefer a multi-line string here for easier editing, reading, and verifying that the indentation is correct!

```Rust
"This crate doesn't yet support compiling on operating systems and architectures other than these:
       - FreeBSD and x86_64
       - FreeBSD and AArch64
       - macOS and x86_64
       - Linux and x86_64
       - Linux and AArch64
       - Android and x86_64
       - Android and AArch64"
```